libs/punchapp/src/lib/config/frameworkConfig.ts

```diff
@@ -32,6 +33,21 @@ export const configure = async (config: IAppConfigurator, c: ComponentRenderArgs
   });
 
   enableAgGrid(config);
+

```

Comment:
Im confused, does the app setup the api clients for the 3d widget or does the widget set it up itself? Either way we need to inject env vars and not hardcode

Reply:
Removed this from the punch app, as it should be implemented in work-order only in this PR